packages/crypto/src/crypto.ts

```diff
@@ -466,3 +568,253 @@ const bigIntToHex = (num: bigint, length: number): string => {
   }
   return hexString.padStart(length, "0");
 };
+
+/**
+ * Verifies a signature from a Turnkey enclave using ECDSA and SHA-256.
+ *
+ * @param {string} enclaveQuorumPublic - The public key of the enclave signer.
+ * @param {string} publicSignature - The ECDSA signature in DER format.
+ * @param {string} signedData - The data that was signed.
+ * @param {Environemnt} environment - An enum PROD or PREPROD to verify against the correct signer enclave key
+ * @returns {Promise<boolean>} - Returns true if the signature is valid, otherwise throws an error.
+ */
+
+const verifyEnclaveSignature = async (
+  enclaveQuorumPublic: string,
+  publicSignature: string,
+  signedData: string
+) => {
+  if (enclaveQuorumPublic != SIGNER_PUBLIC_KEY) {
```

Comment:
This would make it impossible to import into preprod. We might need a way to override this? extra `signerEnclaveOverride?:` param?